components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/internal/FeatureHolder.kt

```diff
@@ -36,6 +50,28 @@ class FeatureHolder<T>(
     fun recordExposure() {
         getSdk()?.recordExposureEvent(featureId)
     }
+
+    fun withCachedValue(value: T?) {
```

Comment:
I'm assuming this is for testing purposes for the apps correct?

Reply:
Added docs.